pr.ts

```diff
@@ -26,16 +26,20 @@ export const changelog = wrap("Require changelog entries on PRs with code change
   const getContentParams = { path: "", owner: pr.head.user.login, repo: pr.head.repo.name }
   const rootContents: any = await danger.github.api.repos.getContent(getContentParams)
 
-  const hasChangelog = rootContents.data.find(file => changelogs.includes(file.name))
-  if (hasChangelog) {
+  const hasChangelog = rootContents.data.find((file: any) => changelogs.includes(file.name))
+  const markedTrivial = pr.title.match("#trivial")
+  if (hasChangelog && !markedTrivial) {
```

Comment:
Just realized something here @ashfurrow 
@orta's suggestion was to `warn` (or just `info` if that exists) when there were changes *but* the issue is trivial. 
Do you want to add that piece as well?

Reply:
Ooooh yeah good idea!